onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -884,7 +884,20 @@ def test_session_with_ortvalue_input(ortvalue):
 
             # The constructed OrtValue should still be valid after being used in a session
             self.assertTrue(np.array_equal(ortvalue2.numpy(), numpy_arr_input))
-            
+
+    def testOrtValue_ghIssue9799(self):
+        if 'CUDAExecutionProvider' in onnxrt.get_available_providers():
+            np.random.seed(42)
```

Comment:
why do we need to set the random seed for this test?

Reply:
Thanks for the review. I just copied it from the repro script. :) I will remove it.